managed/devops/bin/yb_platform_util.py

```diff
@@ -0,0 +1,700 @@
+        """
+        if not self.args.task:
+            sys.stderr.write("Task id required to get task status. Use '-t|--task <task_id>' to pass task_id.")
+            exit(1)
+        self.__task_progress_bar(self.args.task)
+    
+
+    def __parse_arguments(self):
+        """
+        Function to dispaly help message, Add arguments to the python script.
+
+        :return: None
+        """
+        parser = argparse.ArgumentParser(
+        formatter_class = argparse.RawDescriptionHelpFormatter,
+        description = textwrap.dedent('''
+        Script to perform universe actions.
+        --------------------------------
+            1. Get existing universe list. 
+            2. Get existing universe details by universe name in json format.
+            3. Get existing universe details by universe UUID in json format.
+            4. Delete existing universe by universe name.
+            5. Delete existing universe by universe UUID.
+            6. Create a new universe from a json config file.
+            7. Get task progress. 
+            8. Get list of available regions with availability zones. 
+            9. Get list of available providers. 
+
+        Required to export variable:
+        --------------------------------
+            YB_PLATFORM_URL 
+                API URL for yugabyte
+                Example: 
+                    export YB_PLATFORM_URL=http://localhost:9000
+            YB_PLATFORM_API_TOKEN 
+                API token for Yugabyte API
+                Example: 
+                    export YB_PLATFORM_API_TOKEN=e16d75cf-79af-4c57-8659-2f8c34223551
+
+        Actions:
+        --------------------------------
+        get_universe 
+            Get the details of an existing universe as a json file
+            Example: 
+                python yb_platform_util.py get_universe
+
+            Universe json with universe name: 
+                python yb_platform_util.py get_universe -n test-universe
+
+        add_universe |  create_universe
+            Create universe from json file
+            Example: 
+                python yb_platform_util.py create_universe -f test-universe.json -n test-universe-by-name
+
+        del_universe |  delete_universe
+            Delete an existing universe
+            Example: 
+                python yb_platform_util.py delete_universe -n test-universe
+
+        task_status 
+            To get task status
+            Example: 
+                python yb_platform_util.py task_status -t f33e3c9b-75ab-4c30-80ad-cba85646ea39
+
+        get_provider 
+            To get list of available providers
+            Example: 
+                python yb_platform_util.py get_provider
+
+        get_region | get_az 
+            List of available region with availability zones
+            Example: 
+                python yb_platform_util.py get_region
+        ''')
+        )
+        parser.add_argument('actions', type=str, nargs=1, help='Universe actions to perform')
```

Comment:
Is it possible to make this an enum? That way we can maybe also add individual help commands to each action type if required. Does that make sense? Maybe we can do that later. Not really that important.